xknx/devices/cover.py

```diff
@@ -33,12 +33,14 @@
 logger = logging.getLogger("xknx.log")
 
 
+DEFAULT_TRAVEL_TIME_DOWN: Final = 25
+DEFAULT_TRAVEL_TIME_UP: Final = 25
```

Comment:
where is this used?

Reply:
We should move this to the integration if it's not used in the lib

Reply:
Only in init. (I guess for a lot of tests).

Reply:
Ah! Yeah thats fine then :)